tslint.json

```diff
@@ -1,6 +1,11 @@
 {
     "extends": "tslint:latest",
     "rulesDirectory": "built/local/tslint/rules",
+    "linterOptions": {
+        "exclude": [
+            "tests/cases/**/*"

```

Comment:
May be it should be everything in tests folder instead

Reply:
Seems reasonable. Done.